src/thorlabs_cube/aqctl_thorlabs_cube.py

```diff
@@ -60,10 +61,12 @@ def main():
                 dev = TpzSim()
             elif product == "kdc101":
                 dev = KdcSim()
+            elif product == "kpa101":
+                dev = KpaSim()
             else:
-                print(
+                raise ValueError(
                     "Invalid product string (-P/--product),"
-                    " choose from tdc001, tpz001, or kdc101"
+                    " choose from tdc001, tpz001, kpa101, or kdc101"
```

Comment:
temporary manual addition till I get to the dynamic data structure issue for this problem